.github/scripts/Bump.sh

```diff
@@ -16,4 +16,10 @@ done
 echo "."
 
 echo "Updating CHANGELOG.md"
-sed -e "s/^## Unreleased$/## [${new}]/" -i.release "CHANGELOG.md"
\ No newline at end of file
+sed -e "s/^## Unreleased$/## [${new}]/" -i.release "CHANGELOG.md"
+
+echo "Updating tremor dependencies in tremor-script"
+cd tremor-script
+sed -e "s/^tremor-common = { version = \"${old}\"/tremor-common = { version = \"${new}\"/" -i.release "Cargo.toml"
+sed -e "s/^tremor-influx = { version = \"${old}\"/tremor-common = { version = \"${new}\"/" -i.release "Cargo.toml"
+sed -e "s/^tremor-value = { version = \"${old}\"/tremor-common = { version = \"${new}\"/" -i.release "Cargo.toml"
```

Comment:
Really good idea to specify which versions  and include the  crate name, that prevents conflicts 👍  but it might also be needed in some sub crates, not just the upstream `./Cargo.tom` look at the other `Cargo.toml` in the subdirectories to make sure they're updated at well :)

Reply:
Coudl you take a look now?

Reply:
Oh, no, I meant that there is also `tremor-script/Cargo.toml` (and a few others) that will likely need the correct versions inserted and updated :)

Reply:
Oh sorry. Okay so this is actually updating tremor-script/cargo.toml . if you look closely it cd into `tremor-script` , and updates all the dependencies version, that was missing from main (but was in 0.11)

Reply:
I'll check on the others!